node-manager/src/ldkstorage.rs

```diff
@@ -0,0 +1,135 @@
+use std::io;
+use std::io::Cursor;
+use std::ops::Deref;
+use std::sync::Arc;
+
+use bitcoin::BlockHash;
+use lightning::chain::channelmonitor::ChannelMonitor;
+use lightning::chain::keysinterface::{KeysInterface, Sign};
+use lightning::routing::scoring::{ProbabilisticScorer, ProbabilisticScoringParameters};
+use lightning::util::persist::KVStorePersister;
+use lightning::util::ser::{ReadableArgs, Writeable};
+use log::error;
+
+use crate::localstorage::MutinyBrowserStorage;
+use crate::logging::MutinyLogger;
+use crate::node::NetworkGraph;
+
+pub struct MutinyNodePersister {
+    node_id: String,
+    storage: MutinyBrowserStorage,
+}
+
+impl MutinyNodePersister {
+    fn get_key(&self, key: &str) -> String {
+        format!("{}_{}", key, self.node_id)
+    }
+
+    fn read_value(&self, _key: &str) -> Result<Vec<u8>, io::Error> {
```

Comment:
Also why the underscore here `_key` ?

Reply:
I used _key here because we're gonna concatenate the node id to it so and I didn't want people to accidentally use that key when reading if we refractor this eventually